### ISSUE TITLE
Questions on the proof. of Lemma 2.

### ISSUE THREAD

**Issue by libo-huang** [dtuzi123/LifelongVAEGAN#1]
Dear Mr. Ye,

I am happy to meet you at this project through your masterpieces, [Learning Latent Representations Across Multiple Data Domains Using Lifelong VAEGAN](https://link.springer.com/chapter/10.1007/978-3-030-58565-5_46), published in ECCV 2020. But I am confused about the proof of Lemma 2 in the [Appendix](https://static-content.springer.com/esm/chp%3A10.1007%2F978-3-030-58565-5_46/MediaObjects/504476_1_En_46_MOESM1_ESM.pdf). Specifically, it seems there is a type error in Eq. (3). Replace $B_{1^{K-1}}$ with $B_{1^K}$ seems better. If this correct is sound, how can we deduce the following results till Eq. (4)?

Many thanks for your time, and looking forward to your reply.

![b](https://user-images.githubusercontent.com/24219983/128660464-0f3c06a6-0d96-4cc3-be6d-2ae7345c4ea4.jpg)

**Comment by dtuzi123** [comment]
I agree with you, I will replace $B_{1^{K-1}}$ with $B_{1^K}$ in Appendix. For Eq.4, we can sum up all task risks (i=1,...,K) by using the end of Eq.3.
